README.md

```diff
@@ -35,6 +35,9 @@ import { RequestError } from "@octokit/request-error";
 </tbody>
 </table>
 
+> [!NOTE]
```

Comment:
Let me know if using callouts is a good idea.
I used `NOTE` but this could potentially be a `IMPORTANT`

Reply:
```suggestion
> [!IMPORTANT]
```